package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "nodejs-mentoring",
+  "version": "1.0.0",
+  "description": "folder with NodeJs Mentoring 2018Q4 tasks",
+  "main": "src/app.js",
+  "scripts": {
+    "start": "nodemon -r @babel/register ./src/app.js",
+    "build": "npx babel ./src --out-dir dist",
```

Comment:
You don't need to use `npx` in npm scripts.

Reply:
ok, thanks